src/js/components/Menu.js

```diff
@@ -164,6 +164,16 @@ class MenuDrop extends Component {
     return true;
   }
 
+  _renderNestedMenuLinks (menuItems) {
```

Comment:
This doesn't seem to be different to just adding "this.props.children". If menuItems is an Array, we're still returning an Array of components. I suspect I'm missing something though. Can you clarify?